### PR TITLE
Bump stripe version to support Google & Apple pay

### PR DESCRIPTION
**Pull request by odinuge**
Screenshot from 2019-03-18 21-48-10

### CODE REVIEW

File: lego/apps/events/tests/test_payments.py [hunk 1]
```diff
@@ -20,8 +20,8 @@ class StripePaymentTestCase(BaseAPITestCase):
     fixtures = [
         "test_abakus_groups.yaml",
         "test_companies.yaml",
-        "test_events.yaml",
         "test_users.yaml",
+        "test_events.yaml",
```
Comment (edvardvb) [inline lego/apps/events/tests/test_payments.py]: What's up here? Was the previous ordering causing issues when loading fixtures for the tests?

Comment (odinuge) [reply lego/apps/events/tests/test_payments.py]: Yup. `2017/11/13 21:52:34` was the last time we executed the stripe tests :tm:

Comment (edvardvb) [reply lego/apps/events/tests/test_payments.py]: It worked on release, so why would we ever test it again??? 🙃

Comment (odinuge) [reply lego/apps/events/tests/test_payments.py]: Hahaha. Stripe has a quite stable API, so it shouldn't be a huge problem :laughing:

**Pull request merged**